controllers/posts_controller.js

```diff
@@ -1,66 +1,60 @@
-const post = require('../model/posts_model')
-const utils = require('../utils/util')
+const post = require('../model/posts_model');
+const utils = require('../utils/util');
 
-module.exports.addPosts = ({body: {url = null, description = null, tags = null}, user = null}, res) => {
-    let errorMessage = '';
+module.exports.addPosts = async ({body: {url = null, description = null, tags = null}, user = null}, res) => {
+    let errors = [];
 
     if (utils.isEmpty(user.id)) {
-        errorMessage += 'User ID is Null\n'
+        errors.push('User ID is Null');
     }
     if (utils.isEmpty(url)) {
-        errorMessage += 'URL is Null\n'
+        errors.push('URL is Null');
     }
 
-    if (!utils.isEmpty(errorMessage)) {
-        res.status(401).json({
+    if (errors.length > 0) {
+        res.status(400).json({
             error: true,
-            data: errorMessage
+            data: errors.join('\n')
```

Comment:
Good habit IMHO to put commas , for every property (even the last one)

Reply:
that way if you add properties to the object you don't get weird errors cuz you forgot a comma

Reply:
Just seems weird to have a comma after the last value

Reply:
You can put a last comma in objects and also in function signatures FYI